db-init/src/main/resources/database/migrations/V4.1__Create_ExamOrder_Table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS exam_order (
+    id uuid DEFAULT uuid_generate_v4 () NOT NULL,
+    collection_id VARCHAR,
+    status VARCHAR,
```

Comment:
Expecting `status_at timestamp` to be added in another PR to satisfy from the PR description:
> 3. For the /examOrderingStatus endpoint, ~how is the examOrderDateTime field of the payload stored in the DB? If it's not stored, please store it as ordered_at only if the exam was successfully ordered.~ (regardless of status) store the status string and a status_at timestamp.